wordpress-mobile/WordPress-Android
# [Reader Improvements] Post Details bottom actions bar

Part of #19227 

This PR implements the updates to the bottom actions bar inside the Post Details screen to match the new specs. 

The only possibly missing part is updating the icon assets to match the specs (zQnohyMpLzBzQ5jzMMKni3-fi-869:20299#564589863) but we need @osullivanchris to check if that's needed and provide some missing assets in that case. 

**Note:** We will proceed with the code review and merge of this PR even without the confirmation about the icons since this is pointing to the feature branch and we can update it in another PR.

This change is also behind the `ReaderImprovementsFeatureConfig` flag.

## Demo
Flag OFF | Flag ON
--- | ---
![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/472b9d22-49e4-422b-9c03-5662a7fbdfcc) | ![image](https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/f383f052-e258-4092-ac8b-2ff8642f6063) 

## Test
1. Install Jetpack app and login
2. Turn the `ReaderImprovementsFeatureConfig` config off
3. Go to the Reader
4. Open a Post
5. **Verify** the bottom bar in the Post Details still matches the old design
6. **Verify** the bottom bar actions work as expected
7. Turn the `ReaderImprovementsFeatureConfig` config on
3. Go to the Reader
4. Open a Post
5. **Verify** the bottom bar in the Post Details matches the new design
6. **Verify** the bottom bar actions work as expected

## Regression Notes
1. Potential unintended areas of impact
Affect similar action bars in other parts of the app (such as the posts list).

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A.

PR submission checklist:


UI Changes testing checklist:
